zendesk/errors.go

```diff
@@ -0,0 +1,91 @@
+package zendesk
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/amp-labs/connectors/common"
+)
+
+var (
```

Comment:
This list of errors was copied elsewhere and may not be exhaustive. Similarly, it may include errors that don't make sense.